Cargo.toml

```diff
@@ -18,6 +18,8 @@ clap = { version = "4.5", optional = true, features = ["derive"] }
 crc32fast = "1.4.2"
 crossbeam-channel = "0.5.13"
 crossbeam-skiplist = "0.1.3"
+derive_more = { version = "1.0.0", features = ["display"] }
+dotenv = "0.15.0"
```

Comment:
I think we might want to use `dotenvy`. `dotenv`  hasn't been updated in 4 years and `dotenvy` purports to be a well maintained fork that includes some security patches.

https://github.com/allan2/dotenvy